resources/todomvc/big-dom-generator/gen-css.js

```diff
@@ -152,6 +154,7 @@ const getInitialDepth = (element) => {
     return depth;
 };
 
+const cssProperties = [...CSS_PROPERTIES];
```

Comment:
It's not clear to me why we're not using CSS_PROPERTIES directly.

That said I believe this file is removed in the other patch, so these changes won't be used here.